src/tools/clippy/tests/ui/multiple_unsafe_ops_per_block.stderr

```diff
@@ -1,3 +1,35 @@
+error: creating a mutable reference to mutable static is discouraged
+  --> tests/ui/multiple_unsafe_ops_per_block.rs:38:9
+   |
+LL |         STATIC += 1;
+   |         ^^^^^^^^^^^^ mutable reference to mutable static
```

Comment:
Since this test is unrelated, let's fix it (e.g. with `STATIC = STATIC + 1` rather than blessing it.